src/collections/_documentation/platforms/javascript/index.md

```diff
@@ -987,7 +990,43 @@ basic stack trace. This exception is stored here for further data extraction.
 
 : For breadcrumbs created from HTTP requests done via the legacy `XMLHttpRequest` API. This holds the original xhr object.
 
+## Troubleshoot
```

Comment:
This is typically called "Troubleshooting"; I'm not sure I've seen "Troubleshoot" as a heading before.